recipes/fps-uvicorn/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "fps-uvicorn" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jupyter-server/fps/archive/{{ name }}-{{ version }}.tar.gz
+  sha256: 55f9541fe478d0022ceba8a6920c32733a4dfde8c7f34fc10f92cd0791b7fa0e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install ./plugins/uvicorn/ --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - fps-uvicorn = fps_uvicorn.cli:app
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - fps
+    - uvicorn
+    - typer
+
+test:
+  imports:
+    - fps_uvicorn
```

Can you add a test that runs the cli?
```suggestion
    - fps_uvicorn
  - commands:
    - fps-uvicorn
```